src/DownloadAssetsCommand.php

```diff
@@ -7,14 +7,17 @@
 class DownloadAssetsCommand extends Command
 {
     public const REACT_PATH_LOCAL = 'vendor/graphiql/react.production.min.js';
-    public const REACT_PATH_CDN = '//unpkg.com/react@18/umd/react.production.min.js';
+    public const REACT_PATH_CDN = '//unpkg.com/react@17/umd/react.production.min.js';
 
     public const REACT_DOM_PATH_LOCAL = 'vendor/graphiql/react-dom.production.min.js';
-    public const REACT_DOM_PATH_CDN = '//unpkg.com/react-dom@18/umd/react-dom.production.min.js';
+    public const REACT_DOM_PATH_CDN = '//unpkg.com/react-dom@17/umd/react-dom.production.min.js';
 
     public const JS_PATH_LOCAL = 'vendor/graphiql/graphiql.min.js';
     public const JS_PATH_CDN = '//unpkg.com/graphiql/graphiql.min.js';
 
+    public const EXPLORER_PLUGIN_PATH_LOCAL = 'vendor/graphiql/graphiql-plugin-explorer.umd.js';
+    public const EXPLORER_PLUGIN_PATH_CDN = '//unpkg.com/@graphiql/plugin-explorer@0.1.12/dist/graphiql-plugin-explorer.umd.js';
```

Comment:
Given we do not define a version for GraphiQL itself, perhaps we should omit it here too?